packages/react/src/components/Pagination/Pagination.js

```diff
@@ -312,7 +321,7 @@ export default class Pagination extends Component {
     const pageSizes = mapPageSizesToObject(_pageSizes);
 
     return (
-      <div className={classNames} {...other}>
+      <div className={classNames} ref={forwardedRef} {...other}>
```

Comment:
Hmm.. thoughts on doing something like `rest.ref` rather than adding a whole new prop for it in v10, and having to deprecate it in v11 when it's no longer needed?
DatePicker class component did this with `inline` to not have to introduce it as a prop: https://github.com/carbon-design-system/carbon/blob/ddaadaad1e95dd44dc34b5c16268437568123793/packages/react/src/components/DatePicker/DatePicker.js#L340

Reply:
Good call. That is definitely cleaner. I will adjust.